packages/design-picker/src/components/pattern-assembler-cta/index.tsx

```diff
@@ -17,18 +17,18 @@ const PatternAssemblerCta = ( { onButtonClick }: PatternAssemblerCtaProps ) => {
 			<div className="pattern-assembler-cta__image-wrapper">
 				<img className="pattern-assembler-cta__image" src={ blankCanvasImage } alt="Blank Canvas" />
 			</div>
-			<h3 className="pattern-assembler-cta__title">{ translate( 'Start with a blank canvas' ) }</h3>
+			<h3 className="pattern-assembler-cta__title">{ translate( 'Design your own' ) }</h3>
 			<p className="pattern-assembler-cta__subtitle">
 				{ ! isDesktop
 					? translate(
 							"Can't find something you like? Jump right into the editor to design your homepage from scratch."
 					  )
 					: translate(
-							"Can't find something you like? Create something of your own by mixing and matching patterns."
+							"Can't find something you like? Start with a blank canvas and design your own homepage using our library of patterns"
```

Comment:
Looks like it is missing a full stop (`.`) 😄

Reply:
Ooops 🙈